docs/IJSValueWriter-api-windows.md

```diff
@@ -7,78 +7,78 @@ Kind: `interface`
 
 
 
-`IJSValueWriter` is used to write JavaScript values from custom native modules.  It acts as a stream and supports all the types in [`JSValueType`](JSValueType).
+JSON-like stream writer.<br/>It is used to write data that is sent between native modules and the Microsoft.ReactNative.dll.<br/><br/>The JSON-like streams are data structures that satisfy the [JSON specification](https://tools.ietf.org/html/rfc8259). The data structure may have objects with name-value pairs and arrays of items. Property values or array items can be of type `Null`, `Object`, `Array`, `String`, `Boolean`, or `Number`. The `IJSValueWriter` treats the `Number` type as `Int64` or `Double`. See [`JSValueType`](JSValueType).<br/><br/>See the [`IJSValueReader`](IJSValueReader) for the corresponding reader interface.<br/><br/>The [`IJSValueReader`](IJSValueReader) and [`IJSValueWriter`](IJSValueWriter) should be rarely used directly. They are usually used to create serializer and deserializer functions for a type. In the rest of our code we must use these functions to serialize/deserialize our values. We already have a set of serializer and deserializer functions for some standard types.

```

Comment:
> In the rest of our code

Whose code does this refer to? RNW? app code?

Reply:
Updated